proposal/eesss34690/README.rst

```diff
@@ -0,0 +1,87 @@
+=============================================================================
+Proposal for FFT Computing Optimization base on Columnar Array (CA-FFT)
+=============================================================================
+
+Basic information
+=================
+
+The GitHub repository is at: (URL)
+
+Problem to solve
+================
+
+Computing `Fast Fourier Transform <https://en.wikipedia.org/wiki/Fast_Fourier_transform>`__ is very important for digital recording, sampling. It is an algorithm computing the discrete Fourier transform (DFT) of a sequence. Since the sequences usually include lots of data and the computation of FFT is heavy, fasten up the solving speed is crucial for saving caching memory and also optimizing the computing speed.
+
+Currently the most common method to do FFT is ``Cooley-Tukey algorithm``. In this algorithm, it keeps breaking down a DFT of any size :math:`N = m \\times n` into smaller DFTs of sizes :math:`m` and :math:`n`, along with :math:`O(N)` multiplications by twiddle factor (typically either :math:`m` or :math:`n` is a small factor). Therefore, the DFT can be expressed as below:
+
+=============================================================================
+Proposal for FFT Computing Optimization base on Columnar Array (CA-FFT)
+=============================================================================
+
+Basic information
+=================
+
+The GitHub repository is at: (URL)
+
+Problem to solve
+================
+
+Computing `Fast Fourier Transform <https://en.wikipedia.org/wiki/Fast_Fourier_transform>`__ is very important for digital recording, sampling. It is an algorithm computing the discrete Fourier transform (DFT) of a sequence. Since the sequences usually include lots of data and the computation of FFT is heavy, fasten up the solving speed is crucial for saving caching memory and also optimizing the computing speed.
```

Comment:
I assume you want to say

> speed up the solving ...

rather than

> fasten up the solving ...

And "speed up the solution ..." sounds just fine too.

Reply:
I will change the word for expressing it in English. Thank you😊